File: library/src/main/java/com/getstream/sdk/chat/rest/response/MuteUserResponse.java
package com.getstream.sdk.chat.rest.response;

import com.getstream.sdk.chat.model.Mute;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.getstream.chat.android.client.models.User;

public class MuteUserResponse {
    @SerializedName("mute")
    @Expose
    Mute mute;

    @SerializedName("own_user")
    @Expose
    User own_user;

}

File: library/src/main/java/io/getstream/chat/android/client/api/models/MuteUserResponse.kt
package io.getstream.chat.android.client.api.models

import io.getstream.chat.android.client.models.Mute
import io.getstream.chat.android.client.models.User


data class MuteUserResponse(
    var mute: Mute,
    var own_user: User
)
